tools/wheels/upload_wheels.sh

```diff
@@ -41,20 +43,20 @@ set_upload_vars() {
     fi
 }
 upload_wheels() {
-    echo ${PWD}
+    echo "${PWD}"
     if [[ ${ANACONDA_UPLOAD} == true ]]; then
-        if [ -z ${TOKEN} ]; then
+        if [ -z "${TOKEN}" ]; then
```

Comment:
This on the other hand is using single brackets, which is actually a different command altogether. In fact, it's available at `/usr/bin/[` although bash and most other shells include a builtin one for speed purposes. It therefore doesn't include this privileged insight into whether the contained data was a variable expansion, and cannot do the right thing automatically.

In a bash script, there's no reason to ever use `[` instead of `[[`, only use the former if you're writing portable POSIX scripts.

Reply:
My "fixes" didn't help the mac build pass, unfortunately, but I'll amend this commit with your advice.